### PR TITLE
Website changes

### PR DESCRIPTION
**Pull request by cameronoelsen**
-Changed the header styling to match the new styling with the blog that
will be pushed in the coming days

### CODE REVIEW

**Comment by Carreau** [comment]
You don't need to close a PR when you do a mistake, you can force push on the old branch, and the PR will update (if this is the reason you closed #41).

**Comment by cameronoelsen** [comment]
Just pushed some more changes to this branch. Just added the SageMathCloud logo to the front page of the website.

**Comment by cameronoelsen** [comment]
@Carreau are we good to merge this?

**Comment by willingc** [comment]
@cameronoelsen The rendering looks fine. The svg looks odd. Is that really their logo?

**Comment by cameronoelsen** [comment]
@willingc That was the SVG I was supplied so I believe so

**Comment by willingc** [comment]
@cameronoelsen hmm... whelp, if that was what you were given. Perhaps an error in the supplied SVG? Which of the following are you seeing in your build?

screen shot 2015-10-09 at 12 30 24 pm

or

screen shot 2015-10-09 at 12 33 56 pm

**Comment by cameronoelsen** [comment]
screen shot 2015-10-09 at 12 35 30 pm
@willingc This is what Im seeing in my local build...

**Comment by willingc** [comment]
@cameronoelsen well then I would say...@carreau ship it

**Comment by cameronoelsen** [comment]
:shipit:

**Comment by Carreau** [comment]
Sorry I'm just back since yesterday late. Catching up.

File: index.html [hunk 1]
```diff
@@ -203,6 +203,7 @@ <h2 style="text-align:center;">Currently in use at</h2>
                     <a href="https://www.janelia.org/"><li class="col"><img src="assets/janelia.svg" class="greydout" alt="Janelia" height="60" width="178"></li></a>
                     <a href="http://codeneuro.org/"><li class="col"><img src="assets/codeneuro.svg" class="greydout" alt="CodeNeuro" height="60" width="296"></li></a>
                     <a href=""><li class="col"><img src="assets/nsite.svg" class="greydout" alt="NSite" height="50" width="94"></li></a>
+                    <a href=""><li class="col"><img src="assets/sagemathcloud.svg" class="greydout" alt="NSite" height="50" width="94"></li></a>
```
Comment (Carreau) [inline index.html]: Alt is wrong, but it's not blocker.

Comment (willingc) [inline index.html]: @Carreau Eagle eyes.

**Comment by willingc** [comment]
@cameronoelsen @Carreau Please look at the live site. The logo looks wonky to me on Chrome and FF. How does it look to you on a live site?

**Comment by Carreau** [comment]
Yep, does for me too. Will open issue.

**Pull request merged**